server/app/assets/javascripts/util.ts

```diff
@@ -63,7 +64,14 @@ const md = new MarkdownIt({
  *
  * @param {string} text The text to parse into HTML.
  */
-export function formatText(text: string): Element {
+export function formatTextHtml(text: string): Element {
+  const parsedHtml = formatText(text)
+  const html = parser.parseFromString(parsedHtml, 'text/html')
+
+  return html.body
+}
+
+export function formatText(text: string): string {
```

Comment:
I made this helper method because we need a method that returns html in string format for inserting in to the options at `option.innerHtml`. The other applications of markdown (question text and question help text) require an element.